netCDF4/_netCDF4.pyx

```diff
@@ -1421,6 +1508,10 @@ compound types defined for the `netCDF4.Group` or `netCDF4.Dataset` to instances
 variable-length types defined for the `netCDF4.Group` or `netCDF4.Dataset` to instances 
 of the `netCDF4.VLType` class.
 
+**`enumtypes`**: The `enumtypes` dictionary maps the names of
```

Comment:
My preference would be to *avoid* adding another attribute on the Dataset object. I don't think it's necessary or even helpful, given that all that information is in the dtype.